messages.json

```diff
@@ -32,20 +32,24 @@
 			"description": "Explains why there is no general channel.",
 			"title": "No general channel",
 			"text": "There is no general channel in this server. The moderation team does not have time to moderate a general or off-topic channel, nor is it relevant to the server itself. Please read the rules and follow the link to PluralHub there if you are looking for a server for socialization."
-		},
-		{
-			"friendlyName": "Sorting options",
-			"category": "rf",
-			"names": [
-				"sorting",
-				"sort",
-				"ordering"
-			],
-			"description": "Explains sorting options are planned",
-			"title": "Sorting members",
-			"text": "The option to sort members and custom fronts by various things, such as creation date or recent fronters has been suggested before. A feature to add ways to sort is planned, but not any specifics on how it would work."
+		}
+	],
```

Comment:
these [], are wrong, the ], should only come after the very last entry. [] defines a list, {} defines an "object", object being the name, friendly name, category, etc. You can paste your .json file into https://jsonlint.com/ to see if you have any syntax issues.